### PR TITLE
Update test

### PR DESCRIPTION
**Pull request by eric3873**
Update test to pass with Jenkins
**IMPORTANT NOTE: Need to merge nut PR #132 Add mock for TcpClient.**

### CODE REVIEW

**Review by perrettecl — APPROVED** [review]

File: lib/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ etn_target(static ${PROJECT_NAME}-lib
     USES_PRIVATE
         cidr
         nutclient
+        nutclientstub
```
Comment (perrettecl) [inline lib/CMakeLists.txt:17]: Is it needed for the lib or only for test? If it's only for tests, it should not be part of the lib and not be part of the package too. If it's needed also for the runtime package, you need to upgrade the simple image to add the package inside.

**Review by perrettecl — CHANGES_REQUESTED** [review]

File: lib/src/sensor_list.cc [hunk 2]
```diff
@@ -24,6 +24,7 @@
 #include <fty_asset_accessor.h>
 #include <fty_common_nut.h>
 #include <fty_log.h>
+#include <nutclientmem.h>
```
Comment (jeffdegott) [inline lib/src/sensor_list.cc:27]: naming convention and readability?

File: lib/src/sensor_list.h [hunk 3]
```diff
@@ -30,10 +30,10 @@ class Sensors
     explicit Sensors(StateManager::Reader* reader);
     void                                      updateFromNUT(nut::TcpClient& conn);
     bool                                      updateAssetConfig(AssetState::Asset* asset, mlm_client_t* client);
-    void                                      updateSensorList(nut::TcpClient& conn, mlm_client_t* client);
+    void                                      updateSensorList(nut::Client& conn, mlm_client_t* client);
     void                                      publish(mlm_client_t* client, int ttl);
-    void                                      removeInventory(std::string name);
-    bool                                      isInventoryChanged(std::string name);
+    void removeInventory(std::string name);
```
Comment (jeffdegott) [inline lib/src/sensor_list.h:35]: prefer const std::string&

File: lib/tests/sensors.cpp [hunk 4]
```diff
@@ -2,99 +2,170 @@
...
-    fty_proto_set_name(asset, "ups-1");
-    fty_proto_set_operation(asset, FTY_PROTO_ASSET_OP_CREATE);
-    fty_proto_aux_insert(asset, "type", "device");
-    fty_proto_aux_insert(asset, "subtype", "ups");
-    fty_proto_ext_insert(asset, "ip.1", "1.1.1.1");
+    auto& writer = manager.getWriter();
+
+    // [ups-1]: UPS mono
+    fty_proto_t *asset = fty_proto_new (FTY_PROTO_ASSET);
+    fty_proto_set_name (asset, "ups-1");
```
Comment (jeffdegott) [inline lib/tests/sensors.cpp:18]: REQUIRE(asset)  ?

File: packaging/debian/control [hunk 5]
```diff
@@ -17,16 +17,17 @@
 #    with this program; if not, write to the Free Software Foundation, Inc.,
 #    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-Source: fty-nut
-Section: net
-Priority: optional
-Maintainer: fty-nut Developers <eatonipcopensource@eaton.com>
+Source:         fty-nut
```
Comment (jeffdegott) [inline packaging/debian/control:20]: tabs are usefull here?

**Review by jeffdegott — APPROVED** [review]

**Review by aquette — APPROVED** [review]

**Pull request merged**